test/testBALNLSFunctions.jl

```diff
@@ -41,19 +41,19 @@ end
   name, group = get_first_name_and_group(filter_df)
   model = BALNLSModel(name, group)
 
-  @test 4.18565951824972266331e+06 - obj(model, model.meta.x0) ≤ 10e-6
+  @test norm(4.18565951824972266331e+06-obj(model, model.meta.x0)) ≤ 10e-6
```

Comment:
That’s not what I meant; you’re still subtracting… I meant
```
@test 4.18565951824972266331e+06 ≈ obj(model, model.meta.x0)
```

Reply:
Sorry, it's done